Sources/Plasma/FeatureLib/pfPython/cyMisc.cpp

```diff
@@ -504,66 +504,44 @@ ST::string cyMisc::GetClientName(pyKey &avKey)
 
 PyObject* cyMisc::GetAvatarKeyFromClientID(int clientID)
 {
-    PyObject* keyObj = nullptr;
-
     if (clientID == plNetClientMgr::GetInstance()->GetPlayerID())
     {
-        keyObj = pyKey::New(plNetClientMgr::GetInstance()->GetLocalPlayerKey());
+        return pyKey::New(plNetClientMgr::GetInstance()->GetLocalPlayerKey());
     }
     else
     {
-        plNetTransportMember **members = nullptr;
-        plNetClientMgr::GetInstance()->TransportMgr().GetMemberListDistSorted( members );
+        std::vector<plNetTransportMember*> members = plNetClientMgr::GetInstance()->TransportMgr().GetMemberListDistSorted();
 
-        if (members != nullptr)
+        for (plNetTransportMember* mbr : members)
         {
-            for(int i = 0; i < plNetClientMgr::GetInstance()->TransportMgr().GetNumMembers(); i++ )
+            if (mbr != nullptr && mbr->GetAvatarKey() != nullptr && mbr->GetPlayerID() == clientID)
```

Comment:
Can a null member actually be returned?

Reply:
The destructor checks for null elements, so probably not...  But this code was just copied and de-indented, unmodified